### DISCUSSION TITLE
ERROR: schema "pgtle" does not exist

### DISCUSSION THREAD

**Discussion by jp30566347** [supabase/supabase#17994] [Questions] [answered]
Hello folks.

I have issues installing extensions through the supabase-dbdev package manager... it seems the pgtle setup is incorret.

Trying to create extension like this:
```
create extension if not exists pg_tle;
```
But it gives the error `ERROR: schema "pgtle" does not exist`

I suspect my DB infra is too old (created ~April 2023) but I'm not sure how to update it.

Thanks

**Comment by GaryAustin1** [comment, marked as answer]
If you are on a free instance you pause/restore to get the latest updates to software and extensions.  If Pro, you have to request upgrades.
Not commenting on your specific extension or error.

**Reply by jp30566347** [reply]
Thank you. I reached out to support and they will perform the upgrades for us.
